content/documentation.rst

```diff
@@ -0,0 +1,370 @@
+Documentation
+#############
+
+:menu_order: 007
+
+.. contents::
+
+
+General
+=======
+
+**What is documented?**
+
+**For the Chevah project members:**
+
+* Packages, modules, classes and functions using docstrings. 
+
+* Documentation changes are linked to a certain version of SFTPPlus.  When 
+  changes are made, these are anchored to a specific release version.
+
+**For the customers:**
+
+* These can be pre-sales, trial customers and existing customers.
+
+* There are two main types of documentation - technical documentation and the
+  User's Guides section.
+
+* User interfaces that need further explanation and / or guidance.
+
+* User's Guides that are aimed at general audiences.
+  These require more in-depth detail than marketing / promotion materials on
+  the main website. 
+
+* Questions and issues to Support and Sales can instead be answered and
+  linked to using documentation.
+
+* Release notes that explains the changes tied to a specific version.
+
+* Further in-depth notes to help secure file transfer processes using SFTPPlus.
+
+For content that is more marketing / generalized, these will go in the main
+website content.
+
+
+Narrative documentation
+=======================
+
+For narrative documentation use 
+`semantic newlines <http://rhodesmill.org/brandon/2012/one-sentence-per-line/>`_.
+
+Make lines short, and break lines at natural places, such as after commas and
+semicolons, rather than after the Nth column.
+
+..  code-block:: text
+    :linenos:
+
+    When editing a narrative documentation file, I wrap the lines semantically.
+    Instead of inserting a newline at 70 columns (or whatever),
+    or making paragraphs one long line,
+    I put in newlines at a point that seems logical to me.
+    Modern code-oriented text editors are very good at wrapping and arranging
+    long lines.
+
+
+Technical Documentation
+-----------------------
+
+The primary audience are:
+
+- Administrators actively using the software either as an existing customer or
+  on the software trial.
+
+- Decision-makers and implementation of the software.  
+
+Both types of audiences will have varying form of technical backgrounds. 
+
+The documentation is constantly being developed as updates to the product are
+made, or when an issue is found which needs further guidance.
+
+
+User's Guides
+-------------
+
+Pages in the User's Guides are used to describe how a task can be performed by
+applying various configuration options.
+
+This section is also used for other frequent questions sent to Support / Sales
+that are not otherwise covered in the main documentation.
+
+User guides can also be written to the more general audience depending on the
+content.
+
+If the page has a specific audience in mind, state the audience in the
+introduction of the page.
+
+
+Documentation Markup
```

Comment:
I wrote this section particularly aimed at people who are not familiar with the RST format when they first start (like me) or a refresher :)  

So I think it will be useful to mention that this section is specifically about the markup.

Also I think it's clear that this is about narrative documentation (as it's about RST markup) so there is no need to mention it again..